pages/people.md

```diff
@@ -129,6 +129,7 @@ GitHub also has a [contributor list][github-contributors] based on commits.
 | Kenneth McFarland   |                                                                   | [PT][PT]              |
 | Kevin Faro          | [Tetra Concepts LLC][TETRA]                                       | [ET][ET]              |
 | Kyle Van Gilson     |                                                                   |                       |
+| Kartik Sethi        |                                                                   | [IST]                 |
```

Comment:
Can you put this in lexical order above Kenneth's name? Thanks!

Reply:
Also, I just noticed that our "IST" links to "IST-Ireland". But, "IST" can also refer to "IST-India" or "IST-Israel". If you want to add a link that is less ambiguous for your own entry, we can fix other contributors' entries later.